src/java.base/share/classes/java/util/Date.java

```diff
@@ -1012,7 +1012,7 @@ public int hashCode() {
      * <li>{@code mm} is the minute within the hour ({@code 00} through
      *     {@code 59}), as two decimal digits.
      * <li>{@code ss} is the second within the minute ({@code 00} through
-     *     {@code 61}, as two decimal digits.
+     *     {@code 61}), as two decimal digits.
```

Comment:
The doc mentions seconds could be in the range [0, 61]. The normal interval should be [0, 59] but maybe there are situations where leap seconds are inserted (see https://en.wikipedia.org/wiki/Leap_second). Historically, only one second has been used so I wonder why it says "61"?

Reply:
![image](https://github.com/user-attachments/assets/95586ba8-5d20-40de-8ab4-47f88ba048a3)

Reply:
Indeed, given that a leap second can be theoretically inserted or removed at the end of March, June, September and December, it seems strange to have second 61 as well.